fe/fe-core/src/main/java/com/starrocks/service/InformationSchemaDataSource.java

```diff
@@ -455,8 +454,7 @@ private static void genPartitionMetaInfo(long dbId, OlapTable table,
         // REPLICATION_NUM
         partitionMetaInfo.setReplication_num(partitionInfo.getReplicationNum(partition.getId()));
         // DATA_SIZE
-        ByteSizeValue byteSizeValue = new ByteSizeValue(physicalPartition.storageDataSize());
-        partitionMetaInfo.setData_size(byteSizeValue.toString());
+        partitionMetaInfo.setData_size(String.valueOf(physicalPartition.storageDataSize()));
```

Comment:
add comment to explain why here is the String value and then parse the string value back to bigint on BE side.

Reply:
> add comment to explain why here is the String value and then parse the string value back to bigint on BE side.

done.